MTA_limit_trends/Comp/run_comp_grad_data_update.py

```diff
@@ -190,25 +200,30 @@ def update_comp_data(gname, msid_list, eyear, etime):
         mc2 = re.search('comp', gname.lower())
         if mc is not None:
             if mc2 is not None:
-                sub_dir = comp_dir
+                sub_dir = COMP_DIR
             else:
-                sub_dir = grad_dir
+                sub_dir = GRAD_DIR
         else:
-            sub_dir = comp_dir
+            sub_dir = COMP_DIR
 #
 #--- set limit data (a list of lists of limit values)
 #
         alimit   = lim_dict[msid]
 
-        for dtype in ['long', 'short', 'week']:
+        for dtype in DTYPE:
             if dtype == 'long':
                 ofile = msid + '_data.fits'
             else:
                 ofile = msid + '_' + dtype + '_data.fits'
 #
 #--- database file name
 #
-            dfile  = data_dir + gname + '/' + ofile
+            dfile = f"{DATA_DIR}/{gname}/{ofile}"
+#
+#--- If the FULL_GEN option is set to true, then remove this file to fully regenerate it
+#
+            if os.path.isfile(dfile) and FULL_GEN:
+                os.remove(dfile)
```

Comment:
Option to delete the small time section fits file if so desired for testing purposes. by default this is turned off and is only enabled when passing the --full option to the running command.